libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -84,63 +81,6 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
   @Input()
   hasItemSpacing: boolean;
 
-  @Input() useVirtualScroll = false;
```

Comment:
Not related to these lines. 

But in the original PR (#1497) a bunch of comments detailing the role of each input property was removed. 
None of the input properties are well documented in the cookbook. 

So now that we have the file open and we're reverting a bit - should we consider to throw them back in for our own sake? Then we don't have to guess what each input property does based on naming when rebuilding declaratively 😄

[list.component.ts from #1497](
https://github.com/kirbydesign/designsystem/pull/1497/files#diff-98f909a9aa2931c335d0d4cebf4367d6e52652656209d67b1cb21a10fd07024)

Reply:
Excellent suggestion, will do!